packages/millicast-sdk/src/workers/TransformWorker.js

```diff
@@ -52,6 +58,10 @@ addEventListener('message', (event) => {
     case 'insertable-streams-receiver':
       setupPipe(event.data, createReceiverTransform())
       break
+    case 'metadata-sei-user-data-unregistered':
+      uuid = event.data.uuid
```

Comment:
We also need to support how to insert the data, e.g.  insert once/ insert X times / every frame / ....
Current mode is every frame, isn't it?

Reply:
In this case would be each time the function is called